app/assets/javascripts/MedicationAdministered.js

```diff
@@ -0,0 +1,20 @@
+var Schema = mongoose.Schema;
+var Code = Schema.Types.Code;
+var Interval = Schema.Types.Interval;
+var Quantity = Schema.Types.Quantity;
+
+var MedicationAdministeredSchema = new Schema({
+  author_datetime: DateTime,
+  relevant_period: Interval,
+  dosage: Quantity,
+  supply: Quantity,
+  frequency: Code,
+  route: Code,
+  reason: Code,
+  negation_rationale: Code,
+  hqmf_oid: { type: String, default: "2.16.840.1.113883.10.20.28.3.45" },
+  category: { type: String, default: "medication" },
```

Comment:
does it make sense to also have a "type"? What I mean by that in this case is "administered". It could be "ordered", "performed", etc.

Reply:
Thats what `category` basically is, it's how the QDM documentation refers to it anyways. Using `category` also saves us from confusion between using `type` as the literal mongo field type.

Reply:
Yeah, that makes sense. I can't see any use for that off the top of my head, but it's easy enough to do.